src/_includes/related-posts.html

```diff
@@ -4,20 +4,20 @@
 {% from 'macros/util/format/url.html' import slugify as slugify %}
 
 <div class="{{ slugify(header) }}">
-    <h1 class="header-slug">
+    <h2 class="header-slug">
         <span class="header-slug_inner">
             {{ header }}
         </span>
-    </h1>
+    </h2>
     <ul class="list list__unstyled list__spaced">
     {% for similar in more_like_this(post, search_types=doc_types, search_size=quantity) %}
         <li class="list_item">
-            <h4 class='u-mb5'>
+            <h3 class='h4 u-mb5'>
```

Comment:
Yep, I fixed a bunch of instances of single quotes from older code but missed this one. Fixed in  58657c2.